magento/pwa-studio
# bug: noop for fetchLocaleData in storybook config

## Description

storybook failed to run due to a global that wasn't defined

## Related Issue





Closes [PWA-968](https://jira.corp.magento.com/browse/PWA-968).

## Acceptance



### Verification Stakeholders

### Specification



### Verification Steps


1. Run `yarn storybook:venia`. Should load/render without UI error. There will be console errors, but that's expected.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
